test_crash.sh

```
@@ -0,0 +1,37 @@
+fi
+
+echo "Running crash test queries..."
+
+redis-cli FLUSHALL
+if [ $? -ne 0 ]; then
+  echo "Error: FLUSHALL failed"
+  exit 1
+fi
+
+redis-cli GRAPH.QUERY test "MERGE (:label8) MERGE (:label2{})<-[:reltype5]-(node_0{})<-[:reltype7]-({})"
+if [ $? -ne 0 ]; then
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?.

(SC2181)